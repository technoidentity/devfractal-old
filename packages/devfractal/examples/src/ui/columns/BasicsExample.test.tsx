import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Basic } from './BasicsExample'

test('Basics Example of Columns', () => {
  const component: ReactTestRenderer = renderer.create(<Basic />)
  expect(component.toJSON()).toMatchSnapshot()
})
