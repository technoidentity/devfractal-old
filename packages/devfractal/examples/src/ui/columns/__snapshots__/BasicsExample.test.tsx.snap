// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basics Example of Columns 1`] = `
<section
  className="section"
>
  <div
    className="container"
  >
    <h1
      className="title is-4"
    >
      Basic columns
    </h1>
    <div
      className="columns"
    >
      <div
        className="column"
      >
        <div
          className="notification is-primary has-text-centered"
        >
          First column
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="notification is-primary has-text-centered"
        >
          Second column
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="notification is-primary has-text-centered"
        >
          Third column
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="notification is-primary has-text-centered"
        >
          Fourth column
        </div>
      </div>
    </div>
  </div>
</section>
`;
